### ISSUE TITLE
Fix IMDDOS example

### ISSUE THREAD

**Issue by clenk** [oasis-open/cti-documentation#78]
The [IMDDOS Botnet Report](https://www.coresecurity.com/publication/imddos-botnet-discovery-and-analysis) example on the examples page links to a gist. We should move it into this repo and fix/update the content (It's a STIX 2.0 bundle, but it contains a Location object).

**Comment by srrelitz2** [comment]
@clenk @ejratl -- Is this issue still relevant or applicable for where we are now with STIX 2.1 and beyond?

**Comment by ejratl** [comment]
Yes, this example does need to be updated as @clenk describes. You can see it here: https://oasis-open.github.io/cti-documentation/stix/examples It is the last example on the page and it links to a gist, rather than STIX content hosted on the documentation site. It would be nice if this were a 2.1 bundle rather than a 2.0 bundle.
